### DISCUSSION TITLE
I can't submit a findings on CodeHawks

### DISCUSSION THREAD

**Discussion by pymachado** [Cyfrin/security-and-auditing-full-course-s23#111] [Q&A] [answered]
Hi guys. After register to CodeHawks and check my email, I was signed into CodeHawks, however, while I walk through the First Flights and select PuppyRaffle to enter my first finding, I can not submitted it. It does not appear the option to enter the findings. This happens with every project. Actually, I go for projects more recently and it's same.
![Screenshot from 2024-02-03 06-37-31](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/63087258/1d695bfe-765b-46de-80c9-8d341b2f35e7)

**Comment by alymurtazamemon** [comment, marked as answer]
@pymachado This contest has ended, and because of that, you cannot submit new findings. But you can look for vulnerabilities in this code and then check the submitted findings to compare your result.

**Reply by cromewar** [reply]
Yes @alymurtazamemon is right. Please @pymachado check out our upcoming First Flights we would love to see you participating.

**Comment by pymachado** [comment]
Okay,, guys. Thank you for your explanation. It was nice.
